src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Serialization/TagHelperDescriptorJsonConverter.cs

```diff
@@ -62,14 +70,14 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
                         if (reader.Read())
                         {
                             var documentation = (string)reader.Value;
-                            builder.Documentation = documentation;
+                            builder.Documentation = Intern(documentation);
```

Comment:
Currently we "Intern" literally every string that we get out of JSON, but there's CPU overhead costs to doing this, so it makes sense to check which ones we're actually getting something out of. Documentation is saving us memory for sure due to how big the strings are though.